### ISSUE TITLE
Selected State Title Color Not Changing

### ISSUE THREAD

**Issue by onurays** [Yalantis/Segmentio#33]
Please see the code sample and screenshot below. I tried debugging `Segmentio` and `SegmentioCell` classes and seems like selected state is never set true. Because, `UICollectionViewDataSource` is not triggered after selection changed from `reloadSegmentio()`.

<img width="374" alt="screen shot 2016-12-14 at 14 23 57" src="https://cloud.githubusercontent.com/assets/1254401/21180187/78820e7e-c200-11e6-805a-dfb1abc30c3d.png">

```Swift
var content = [SegmentioItem]()
        
        content.append(SegmentioItem(
            title: "ONE WAY",
            image: nil))
        content.append(SegmentioItem(
            title: "ROUNDTRIP",
            image: nil))
        
        
        switchTripType.setup(
            content: content,
            style: SegmentioStyle.onlyLabel,
            options: SegmentioOptions(
                backgroundColor: UIColor.clear,
                maxVisibleItems: 3,
                scrollEnabled: true,
                indicatorOptions: SegmentioIndicatorOptions(
                    type: .bottom,
                    ratio: 1,
                    height: 2,
                    color: UIColor.white
                ),
                
                horizontalSeparatorOptions: SegmentioHorizontalSeparatorOptions(
                    type: SegmentioHorizontalSeparatorType.topAndBottom,
                    height: 1,
                    color: UIColor.clear
                ),
                verticalSeparatorOptions: nil,
                /*
                verticalSeparatorOptions: SegmentioVerticalSeparatorOptions(
                    ratio: 0.6,
                    color: UIColor.white
                ),
                */
                imageContentMode: UIViewContentMode.center,
                labelTextAlignment: NSTextAlignment.center,
                labelTextNumberOfLines: 1,
                segmentStates: SegmentioStates(defaultState:
                    SegmentioState(
                        backgroundColor: UIColor.clear,
                        titleFont: UIFont.systemFont(ofSize: UIFont.smallSystemFontSize),
                        titleTextColor: UIColor.white)
                    ,selectedState: SegmentioState(
                        backgroundColor: UIColor.clear,
                        titleFont: UIFont.boldSystemFont(ofSize: UIFont.smallSystemFontSize),
                        titleTextColor: UIColor.white.withAlphaComponent(0.30))
                    ,highlightedState: SegmentioState(
                        backgroundColor: UIColor.white.withAlphaComponent(0.15),
                        titleFont: UIFont.boldSystemFont(ofSize: UIFont.smallSystemFontSize),
                        titleTextColor: UIColor.white)),
                animationDuration: 0.1
            )
        )
```

**Comment by onurays** [comment]
Here is my solution & suggestion:

Segmentio.swift:
`
 public func reloadSegmentio() {
        segmentioCollectionView?.collectionViewLayout.invalidateLayout()
        segmentioCollectionView?.reloadData()
        scrollToItemAtContext()
        moveShapeLayerAtContext()
    }
`

This issue is related to the iOS 10 update of UICollectionView. `collectionViewLayout.invalidateLayout()
` is not triggering fresh setup of the view. So reloading data seems a good solution to me.

Lastly, I changed `selectedSegmentioIndex = -1` to `selectedSegmentioIndex = 0`. Because;

1. There is another bug which seems tab is selected, but again not setting fonts etc.
2. Default selection of the first item is a must / good feature for my opinion.

**Comment by maxcampolo** [comment]
Hey @onurays, this is fixed in PR #32! If you're using Cocoapods you will need to point to master for now. Also, if there are no items, `selectedSegmentioIndex` is not equal to 0 so it must be -1.

**Comment by jarrillaga** [comment]
Hi @onurays, how can I update my pod to have this fix? Thanks, J

**Comment by onurays** [comment]
Hi @jarrillaga, did you try `pod update`?

**Comment by serejahh** [comment]
Fixed in #32
